src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,31 @@
 import React from 'react';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
+import { PostComment } from '../../types/PostComment';
+import './CommentInfo.scss';
+
+type Props = {
+  comment: PostComment;
+};
+
+export const CommentInfo: React.FC<Props> = ({ comment }) => (
```

Comment:
It will be good to make a destructive assignment

Reply:
agree